package.json

```diff
@@ -1569,12 +1557,6 @@
                     "description": "%jupyter.configuration.jupyter.jupyterLaunchRetries.description%",
                     "scope": "resource"
                 },
-                "jupyter.jupyterCommandLineArguments": {
```

Comment:
Based on telemetry this is not used, hence this is being deprecated, will remove the code that uses this setting in a later release.